test/modules/tilecache.js

```diff
@@ -314,200 +314,200 @@
-            for (let tile of tileCache._tilesLoaded) {
-                const cache = tile.getCache();
-                test.equal(cache.type, T_B, "Main Cache Updated (suite 2).");
-                test.equal(cache.data, newValue, "Main Cache Updated (suite 2).");
-            }
-
-            // Now test whether data reset works, value 1 -> copy perfomed due to internal cache cration
-            viewer.removeHandler('tile-invalidated', testHandler);
-            testHandler = async e => {
-                const data = await e.getData(T_B);
-                test.equal(data, 1, "Copy: creation of a working cache.");
-                await e.setData(-8, T_E);
-                e.resetData();
-            };
-            viewer.addHandler('tile-invalidated', testHandler);
-            await viewer.world.requestInvalidate(true);
-            await sleep(1);  // necessary to make space for a draw call
-            testDrawingRoutine(2); // Value +2 rendering from original data
-
-            for (let tile of tileCache._tilesLoaded) {
-                const origCache = tile.getCache(tile.originalCacheKey);
-                test.ok(tile.getCache() === origCache, "Main cache is now original cache.");
-            }
-
-            // Now force main cache creation that differs
-            viewer.removeHandler('tile-invalidated', testHandler);
-            testHandler = async e => {
-                await e.setData(41, T_B);
-            };
-            viewer.addHandler('tile-invalidated', testHandler);
-            await viewer.world.requestInvalidate(true);
-
-            // Now test whether data reset works, even with non-original data
-            viewer.removeHandler('tile-invalidated', testHandler);
-            testHandler = async e => {
-                const data = await e.getData(T_B);
-                test.equal(data, 42, "Copy: 41 + 1.");
-                await e.setData(data, T_E);
-                e.resetData();
-            };
-            viewer.addHandler('tile-invalidated', testHandler);
-            await viewer.world.requestInvalidate(false);
-            await sleep(1);  // necessary to make space for a draw call
-            testDrawingRoutine(42);
-
-            for (let tile of tileCache._tilesLoaded) {
-                const origCache = tile.getCache(tile.originalCacheKey);
-                test.equal(origCache.type, T_A, "Original cache data was not affected, the drawer uses main cache even after refresh.");
-                test.equal(origCache.data, 0, "Original cache data was not affected, the drawer uses main cache even after refresh.");
-            }
-
-            test.ok(testTileCalled, "Drawer tested at least one tile.");
-            done();
-        });
-        viewer.open([
-            {isTestSource: true},
-            {isTestSource: true},
-        ]);
-    });
+    // QUnit.test('Tile: basic rendering & test setup', function(test) {
```

Comment:
Don't forget to uncomment this and fix it up before we merge :)

Reply:
I won't :)